docs/user-manual.md

```diff
@@ -1,229 +1,263 @@
 
-This is an example of the `RunnerPool` custom resource.
+Here is an example of the RunnerPool resource.
 
 ```yaml
 apiVersion: meows.cybozu.com/v1alpha1
 kind: RunnerPool
 metadata:
   name: runnerpool-sample
+  namespace: <your RunnerPool namespace>
 spec:
-  repositoryName: repository-sample
+  repositoryName: "<your Repository>"
   replicas: 3
   slackAgent:
-    serviceName: slack-agent
+    serviceName: slack-agent.meows.svc
 ```
 
-The controller creates a `Deployment` based on `template` in `RunnerPool`.
-It modifies `template` just to satisfy the minimal requirement to run GitHub Actions.
-
-The controller is mainly responsible for:
-
-- Add the GitHub organization name specified with the controller CLI's option and
-  the repository name defined in the `RunnerPool` manifest to `metadata.labels`.
-- Add environment variables needed to run `entrypoint`.
-  `entrypoint` is a default command for the runner container and contains
-  [`github.com/actions/runner`](https://github.com/actions/runner).
-
-You are responsible for:
+When you create the above RunnerPool, meows creates three runner pods (the number is specified by `.spec.replicas`) in `<your RunnerPool namespace>`.
+Then, meows registers each runner pod as a repository-level runner in the specified repository(`.spec.repositoryName`).
 
-- `spec/slackAgent/serviceName` is a `Service` name that can be resolved inside a
-  Kubernetes cluster, so a `Service` name is acceptable. If `spec/slackAgent/serviceName`
-  is omitted, the `Pod`s do not send notifications to Slack.
+After running pods, you can check whether the runners are registered to GitHub on the **Actions** page under each repository's **Settings**.
+E.g. `https://github.com/<your Organization>/<your Repository>/settings/actions/runners`
 
-After running `Pod`s, you can check if the runners are actually registered to
-GitHub on the **Actions** page under each repository's **Settings**
-(e.g. https://github.com/cybozu-go/meows/settings/actions).
+## Writing Workflow
 
-How to use self-hosted runners
-------------------------------
+There are a few tips using runners registered by meows.
 
-meows provides the following commands, you have to execute these commands in your workflow.
-
-- `job-started`
-    - Notify a runner pod that a workflow has been started.
-      You need to call this command at the start of the workflow.
-
-- `job-success`, `job-cancelled`, `job-failure`
-    - Notify a runner pod that the result of a workflow.
-      You need to call these commands at the end of the workflow with [Job status check functions](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions).
+1. Runners have a specific label determined from the name and namespace of the RunnerPool.
+   - The format is `<your RunnerPool Namespace>/<your RunnerPool Name>`.
+2. Some commands should be called in jobs.
```

Comment:
I think this description is ambiguous. (who calls? some or all?)
It should be clearly written that users must call these commands in their workflows.